symfony/src/Service/ParticipantSummaryService.php

```diff
@@ -19,7 +20,8 @@ public function getParticipantById($participantId)
     {
         try {
             $response = $this->api->get(sprintf('rdr/v1/Participant/%s/Summary', $participantId));
-            return json_decode($response->getBody());
+            $participant = json_decode($response->getBody()->getContents());
+            return new Participant($participant);
```

Comment:
Quick note here to make sure we test other implementations using `ParticipantSummaryService::getParticipantById()` (thinking mostly of deceased reports), as I believe those might be expecting the JSON object instead of the entity. In hindsight, it was my mistake not to do it this way to begin with.

Reply:
Thanks for checking. Actually the `Participant` class just returns the same JSON object structure with some additional properties attached to it. I noticed this before and made sure it didn't break other implementations.